Applications/web_portal/web_app/tests/login_page_tester/source/index.tsx

```diff
@@ -1,8 +1,60 @@
+import {css, StyleSheet} from 'aphrodite';
+import * as Beam from 'beam';
 import * as React from 'react';
 import * as ReactDOM from 'react-dom';
 import * as WebPortal from 'web_portal';
 
+/** The React state for the TestApp. */
```

Comment:
Properties are declared first, then State.

Remove documentation from the State class (it's an internal/private class).